.github/workflows/build.yml

```diff
@@ -43,3 +43,15 @@ jobs:
             ./lib/ryzenadj.h
             ./prebuilt/WinRing0x64.dll
             ./prebuilt/WinRing0x64.sys
+
+  Linux:
+    runs-on: Ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Dependencies
+      run: sudo apt install libpci-dev
+
+    - name: run-cmake
+      uses: lukka/run-cmake@v3.3
```

Comment:
Does not upload linux bin, because it is unnecessary. If you like a linux artifact, I could just add it.